includes/class-sensei-modules.php

```diff
@@ -1113,11 +1120,11 @@ class="wrap <?php echo esc_attr( $this->order_page_slug ); ?>">
 								if ( isset( $_GET['course_id'] ) ) {
 									$course_id = intval( $_GET['course_id'] );
 									if ( $course_id > 0 ) {
-										$modules = $this->get_course_modules( $course_id );
+										$modules = Sensei_Course_Structure::instance( $course_id )->get_modules();
```

Comment:
Under the hood, the `get_modules` will use the `get_course_modules`. But I refactored that to consider the `Sensei_Course_Structure`, the source of truth for this type of information.